### DISCUSSION TITLE
alist+aliyun+rclone 无法打开文件或播放视频

### DISCUSSION THREAD

**Discussion by Jue-Xu** [AlistGo/alist#1402] [Q&A] [answered]
我目前在alist添加了阿里云盘，在alist的网页端能够正常播放阿里云盘内的视频。但是通过rclone挂载（webdav）到NAS（linux）本地后，阿里云盘的文件目录能够在NAS上正确显示，但是无法打开阿里云盘内的文件或播放视频（具体来说，在emby中添加了阿里云盘的文件夹，可以识别到视频，但是无法播放）。同样方法在alist下挂载的Google drive则没有类似的问题。
希望有经验的朋友帮助

**Comment by nitsort** [comment]
rclone挂载阿里云盘也许是Referer问题，参考 #630 ，或者贴一下alist和rclone的日志看看。

**Reply by Jue-Xu** [reply, marked as answer]
> rclone挂载阿里云盘也许是Referer问题，参考 #630 ，或者贴一下alist和rclone的日志看看。

参考 https://github.com/alist-org/alist/discussions/630 的方法，Rclone 加上--header "Referer:" 完美解决！太感谢了，困惑好久了。

另外，alist的log在哪里？文档里没找到

**Comment by Jue-Xu** [comment]
已解决！

**Reply by tzzzer** [reply]
如何解决的

**Reply by Jue-Xu** [reply]
Rclone 命令 加上--header "Referer:" 参数

**Reply by 9fav** [reply]
请问大神可以提供一下相信的添加命令和参数的方法吗？本人是业余爱好者，知识比较欠缺

**Reply by Jue-Xu** [reply]
翻一下帖子吧

**Comment by aTenb** [comment]
好奇怪啊，我以前一直没有这个问题你也是挂到emby正常播放。3.11.0那个时候。然后有1个多月没用了，今天升级到3.16.x，开始出现这个问题。这是啥引起的，大佬你半年前就遇到了？

**Reply by Jue-Xu** [reply]
Rclone 命令 加上--header "Referer:" 参数
